modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1127,6 +1127,27 @@ public void testAddObjectEntry() throws Exception {
 			).put(
 				"name", listTypeEntry.getName(LocaleUtil.US)
 			).build());
+
+		// Relationship
+
+		String newExternalReferenceCode = RandomTestUtil.randomString();
```

Comment:
@carolmariaabb  When I try this name, happens a mismatch of the variables name. Because into the `new ObjectEntry()` the "externalReferenceCode" recognized is the attribute of the instance itself, instead of this methods variable